Formula/unit.rb

```diff
@@ -0,0 +1,60 @@
+class Unit < Formula
+  desc "Dynamic web and application server"
+  homepage "https://unit.nginx.org"
+  url "https://unit.nginx.org/download/unit-1.15.0.tar.gz"
+  sha256 "f8b231d02865b5695825a2a77ecbfd0cff33c0954c0d351d2d773d7e5f6237dd"
+  head "https://hg.nginx.org/unit", :using => :hg
+
+  depends_on "openssl@1.1"
+
+  def install
+    system "./configure",
+              "--openssl",
+              "--cc-opt=-I#{Formula["openssl@1.1"].opt_prefix}/include",
+              "--ld-opt=-L#{Formula["openssl@1.1"].opt_prefix}/lib",
+              "--prefix=#{prefix}"
+
+    system "./configure", "python"
```

Comment:
I'm going to assume this uses python. In which case, can you add `depends_on "python@3.8"` and see if that works?

Reply:
Actually, Unit have modules for different languages: python (both 2 & 3), php, perl, ruby, java, node, go. 
I've test it with system Python to minimise dependencies.

Yes, it can be build with modules for system python and python@3.8 at the same time.

If it makes sense, I can add dependencies for python@3.8, perl, ruby, java. Node & go modules can be installed via `npm` and `go` tools. PHP module on Mac OS is more tricky, it requires to rebuild PHP with embed support, so it is too complicated to add it now.

Or modules should be a a separate formulas? In this case I need an advise or sample to look at. Unit module is a shared library linked against language runtime, which needs to be placed in a `modules` Unit directory.

Reply:
I've added python, perl, ruby and java modules to formula. It works, but I don't like the result because it is _solid_ and depends on too many other packages. Usually, people need 1-2 modules, not everything.
Could you please advise how to split it (or point similar existing formula).

Reply:
I wasn't suggesting that and I'd rather not do this. Just if you're shipping python, make sure you depend on the latest python (preferably 3.8). If people want more modules they'll just have to submit a pull request.

Reply:
May I depend on python provided by macOS to minimise extra dependencies?

Reply:
That's only python 2, which is EOL and shouldn't be used unless the software has no other way. So no.

Reply:
Ok. I would better remove modules and dependencies from the main package and provide separate formula(s) when required.